### DISCUSSION TITLE
几天了我蓝灯还是无法登录 #774

### DISCUSSION THREAD

**Discussion by pppppppppppppjsjns** [getlantern/lantern-forum-cn#775] [Q&A]
几天了我蓝灯还是无法登录 #774

重新登陆不发邮箱验证码
帐号574471458@qq.com
您尾号6381交行信用卡05日00时06分成功消费人民币344.00

**Comment by MaxMKMax** [comment]
@pppppppppppppjsjns 
您好. 请您在此提供我支付单据. 我再次帮您提交问题. 不好意思让您久等了. 谢谢您的耐心等待!

您是看能把支付单据发到这里吗? 如果不能就发邮箱给我. 谢谢! 
mike@getlantern.org

您之前的图被GitHub屏蔽了. 我这里看不到. 还是发邮箱给我吧!
